### PR TITLE
Refinement, typos, ...

### PR DESCRIPTION
**Pull request by ChristianMayer**

### CODE REVIEW

File: doc/manual/de/config/widgets/index.rst [hunk 1]
```diff
@@ -120,9 +120,9 @@ Sonstige Widgets
...
 +=================================================+=============================================================================================================================================================================================================================================================================================================================================================+
-| :doc:`Break <break/index>`                      | Fügt der Visu einen Zeilenumbruch hinzu                                                                                                                                                                                                                                                                                                                     |
+| :doc:`Break <break/index>`                      | Fügt der Visu einen Zeilenumbruch hinzu.                                                                                                                                |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: last Table delimiter | not in line with the ones above/below

File: doc/manual/de/config/widgets/index.rst [hunk 2]
```diff
@@ -120,9 +120,9 @@ Sonstige Widgets
...
-| :doc:`Group <group/index>`                      | Mit group können Widgets in einem Rahmen als Gruppe zusammengefasst werden. Der Gruppe kann über "layout" die Größeneigenschaft mitgeben werden sowie alle zur Verfügung stehenden Widgets zuordnen.                                                                                                                                                        |
+| :doc:`Group <group/index>`                      | Mit group können Widgets in einem Rahmen als Gruppe zusammengefasst werden. Der Gruppe kann über "layout" eine Größe eingestellt werden.                                |
 +-------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: last Table delimiter | not in line with the ones above/below

Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: Der letzte Satz klingt irgendwie komisch, besser sowas wie: "Die Größe der Gruppe kann über "layout" eingestellt werden."

File: doc/manual/en/config/widgets/index.rst [hunk 3]
```diff
@@ -22,33 +22,33 @@ actions, e.g. you can switch on a lamp by clicking a switch-widget, or you can n
...
-| :doc:`Refresh <refresh/index>`                  | With the widget refresh, the Visu is added a switch, which allows the Visu to be reloaded in the browser?                                                                                                                                                                                                 |
+| :doc:`Refresh <refresh/index>`                  | With the widget refresh, the visu is added a switch, which allows the visu to reloaded the displayed data                                                                                                                                                                                                 |
 +-------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: "to reload"

File: doc/manual/en/config/widgets/index.rst [hunk 4]
```diff
@@ -22,33 +22,33 @@ actions, e.g. you can switch on a lamp by clicking a switch-widget, or you can n
...
-| :doc:`Reload <reload/index>`                    | With the reload widget, the Visu is added a switch, which allows to reload the Visu in the browser?                                                                                                                                                                                                       |
+| :doc:`Reload <reload/index>`                    | With the reload widget, the visu is added a switch, which allows to reload the full visu in the browser                                                                                                                                                                                                   |
 +-------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: please add a "." to the end of the line

File: doc/manual/en/config/widgets/index.rst [hunk 5]
```diff
@@ -22,33 +22,33 @@ actions, e.g. you can switch on a lamp by clicking a switch-widget, or you can n
...
-| :doc:`Switch <switch/index>`                    | Adds a button to the Visu. Switching actions can be triggered. The switch is typically used for ON/OFF actions. (E.g., light, sockets, etc.)                                                                                                                                                              |
+| :doc:`Switch <switch/index>`                    | Adds a button to the visu. Switching actions can be triggered. The switch is typically used for ON/OFF actions. (E.g. light or power sockets)                                                                                                                                                             |
 +-------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."?

File: doc/manual/en/config/widgets/index.rst [hunk 6]
```diff
@@ -81,21 +81,21 @@ These widgets have a purely informative function, an operation is not possible.
...
-| :doc:`Image <image/index>`                        | Adds a static image or a camera image to the visu.                                                                                    |
+| :doc:`Image <image/index>`                        | Adds a static image or a live camera image to the visu.                                                                               |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Info <info/index>`                          | Adds a dynamic field to the Visu that can represent values from the BUS. (e.g. 14-byte text or counter headers)                       |
+| :doc:`Info <info/index>`                          | Adds a dynamic field to the visu that can represent values from the BUS. (e.g. 14-byte text or temperature measurements)              |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."?

File: doc/manual/en/config/widgets/index.rst [hunk 7]
```diff
@@ -81,21 +81,21 @@ These widgets have a purely informative function, an operation is not possible.
...
-| :doc:`Line <line/index>`                          | Adds a horizontal line to the visu to divide the visual side into individual areas.                                                   |
+| :doc:`Line <line/index>`                          | Adds a horizontal line to the visu to divide the display into individual areas.                                                       |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`RGB <rgb/index>`                            | With the RGB widget you can display a color in the Visu.                                                                              |
+| :doc:`RGB <rgb/index>`                            | With the RGB widget you can display a colour in the visu.                                                                             |
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: not if we want to use american english or british english regarding color vs. colour. I would tend to use color, but I am not sure how its handled throughout the docs.

File: doc/manual/en/config/widgets/index.rst [hunk 8]
```diff
@@ -81,21 +81,21 @@ These widgets have a purely informative function, an operation is not possible.
...
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Web <web/index>`                            | Adds an area to the Visu, where external websites can be displayed.                                                                   |
+| :doc:`Web <web/index>`                            | Adds an area to the visu, where external websites can be displayed.                                                                   |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`WG_Plugin_Info <wgplugininfo/index>`        | Tbd.                                                                                                                                  |
+| :doc:`WG_Plugin_Info <wgplugininfo/index>`        | Adds an dynamic field to the viso that shows live information from a WireGate plugin.                                                 |
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: "visu"

File: doc/manual/en/config/widgets/index.rst [hunk 9]
```diff
@@ -118,15 +118,15 @@ Other widgets
...
-| :doc:`Include <include/index>`                    | Allows you to split the visu_config.xml into multiple files and re-merge the parts using * include *.                                    |
+| :doc:`Include <include/index>`                    | Allows you to split the visu_config.xml into multiple files and re-merge the parts using *include*.                                      |
 +---------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Navbar <navbar/index>`                      | With the widget navbar, you can add a navigation menu to the entire Visu. The menu can be displayed on a page (top, bottom, left, right) |
+| :doc:`Navbar <navbar/index>`                      | With the widget navbar, you can add a navigation menu to the entire visu. The menu can be displayed on a page (top, bottom, left, right) |
 +---------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."

File: doc/manual/en/config/widgets/index.rst [hunk 10]
```diff
@@ -162,29 +162,29 @@ For example:
...
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
-| :doc:`Clock <plugins/clock/index>`                                    | Widget to display the time of a GA. (e.g. The WireGate)                                                               |
+| :doc:`Clock <plugins/clock/index>`                                    | Widget to display the time of an address.                                                                             |
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
-| :doc:`SVG <plugins/svg/index>`                                        | Scalable vector graphics?                                                                                             |
+| :doc:`SVG <plugins/svg/index>`                                        | Demo plugin to show the integration of an interactive scalable vector graphic                                         |
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."

File: doc/manual/en/config/widgets/index.rst [hunk 11]
```diff
@@ -162,29 +162,29 @@ For example:
...
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
-| :doc:`Timeout <plugins/timeout/index>`                                | Timeout                                                                                                               |
+| :doc:`Timeout <plugins/timeout/index>`                                | Changes currently displayed page after some time of inactivity                                                        |
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."

File: doc/manual/en/config/widgets/index.rst [hunk 12]
```diff
@@ -162,29 +162,29 @@ For example:
...
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
 | :doc:`UPNPController <plugins/upnpcontroller/index>`                  | UPNP Controller                                                                                                       |
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
-| :doc:`Speech <plugins/speech/index>`                                  | Uses the browsers builtin text-to-speech engine to speak text                                                         |
+| :doc:`Speech <plugins/speech/index>`                                  | Uses the browsers built in text-to-speech engine to speak text                                                        |
 +-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+
```
Comment (peuter) [inline doc/manual/en/config/widgets/index.rst]: missing "."

File: doc/manual/en/config/widgets/index.rst (hunk 7 above)
Comment (ChristianMayer) [inline doc/manual/en/config/widgets/index.rst]: Well, I tend to stick to the orignial, to the BE. But this already created an inconsistency with the ColorChooser as having that name and use `colour` in the description seemed to be strange...

So far I also have no answer to that question

**Pull request merged**